src/MAUI/Maui.Samples/Samples/Data/EditFeaturesUsingFeatureForms/readme.md

```diff
@@ -0,0 +1,53 @@
+# Edit features using feature forms
+
+Display and edit feature attributes using feature forms.
+
+![Image of Edit features using feature forms sample](EditFeaturesUsingFeatureForms.jpg)
+
+## Use case
+
+Feature forms help enhance the accuracy, efficiency, and user experience of attribute editing in your application. Forms can be authored as part of the web map using [Field Maps Designer](https://www.arcgis.com/apps/fieldmaps/) or using Map Viewer. This allows a simplified user experience to edit feature attribute data on the web map.
+
+## How to use the sample
+
+Tap a feature on the map to open a sheet displaying the feature form. Select form elements in the list and perform edits to update the field values. Tap the submit icon to commit the changes on the web map.
+
+## How it works
+
+1. Add a `Map` to the `MapView` using `Portal` URL and item ID.
+2. When the map is tapped, perform an identify operation to check if the tapped location is an `ArcGISFeature`.
+3. Create a `FeatureForm` object using the identified `ArcGISFeature`.
+  * **Note:** If the feature's `FeatureLayer`, `ArcGISFeatureTable`, or the `SubtypeSublayer` has an authored `FeatureFormDefinition`, then this definition will be used to create the `FeatureForm`. If such a definition is not found, a default definition is generated.
+4. Use the `FeatureForm` toolkit component to display the feature form configuration by providing the created `featureForm` object.
+5. Optionally, you can add a `ValidationErrorVisibility` option to the `FeatureForm` toolkit component that determines the visibility of validation errors.
+6. Once edits are added to the form fields, check if the validation errors list are empty using `featureForm.ValidationErrors` to verify that there are no errors.
+7. To commit edits on the service geodatabase:
+    1. Call `featureForm.FinishEditing()` to save edits to the database.
```

Comment:
`FinishEditingAsync()`

I think for non-static methods, in readmes, we just use the method name.